Livecheckables/jdnssec-tools.rb

```diff
@@ -0,0 +1,4 @@
+class JdnssecTools
+  livecheck :url   => "https://github.com/dblacka/jdnssec-tools/releases",
+            :regex => %r{href=".*jdnssec-tools-(\d+(?:\.\d+)+)}i
```

Comment:
```suggestion
  livecheck :url   => "https://github.com/dblacka/jdnssec-tools.git",
            :regex => /^v?(\d+(?:\.\d+)+)$/
```

As mentioned previously, we want to avoid using release pages whenever possible. Release pages are paginated and only display a limited number of releases (where pre-release versions may push the latest stable version onto a different page), so it's better to use the Git repo tags when we're simply looking for release tags. [There are times where we do use the `.../releases/latest` URL but this isn't one of those times.]

Matching against the Git repo tags, we can just use the standard regex for Git repo tags instead.